internal/buildscripts/packaging/release/helpers/util.py

```diff
@@ -528,9 +528,9 @@ def download_github_assets(github_release, args):
     for asset in github_release.get_assets():
         ext = os.path.splitext(asset.name)[-1].strip(".")
         if asset.name == "checksums.txt":
-            checksums_asset = Asset(url=asset.browser_download_url)
+            checksums_asset = Asset(url=asset.url, name=asset.name)
```

Comment:
Could be helpful to describe why this change is made in the commit message or as a comment